README.md

```diff
@@ -1,2 +1,62 @@
-# proposals
-This repository holds all the past and current proposals for the Mixxx Software
+# Proposals
+
+This repository holds all the past and current proposals for the Mixxx Software. It’s the single place for
+reviewing, discovering, and working on the design documents. It is also a record of past decisions and approvals.
+
+## Current Proposals
+
+* The
+  [open PRs with the `proposal` label](https://github.com/acolombier/mixxx-proposals/pulls?q=is%3Aopen+is%3Apr+label%3Aproposal)
+  show all the pending proposals.
+* The [proposals directory](./proposals) shows all the accepted proposals. See the “Implementation Status” for details
+  on the implementation.
+* The
+  [PRs with the `proposal` label that are closed without merging](https://github.com/acolombier/mixxx-proposals/pulls?q=is%3Apr+label%3Aproposal+is%3Aclosed+is%3Aunmerged)
+  show all the rejected proposals.
+
+## What’s a Design Document?
+
+It’s essential to clearly explain the reasons behind certain design decisions to have a community consensus. This is
+especially important in Mixxx, where every decision might have a significant impact given the high adoption and
+stability of the software and standards we work on.
+
+In our world, no decision is perfect, so having a design document explaining our trade-offs is essential.
+Such a document can also be used later as a reference and for knowledge-sharing purposes.
+
+Design documents do not always reflect what has been (or will be) implemented. Implementation details
+might have changed since a feature was merged. Design docs are not considered documentation and can not define a
+standard.
+Instead, it should explain the motivation, scope, decisions, and alternatives considered.
+
+A design document shall describe the use-cases the proposal will implement and also the use-cases that are explicitly
+not included.
+
+It may also contain requirements and acceptance criteria of a derived Pull Request. This avoid feature creep and adjust
+the expectations during a review.
+
+## Proposal Process
+
+Don’t get scared to propose ideas! It’s amazing to innovate in the open and get feedback on ideas.
+
+The process of proposing a change via a design document is the following:
+
+1. Fork `github.com/acolombier/mixxx-proposals`.
+2. Create a GitHub Pull Request with a design document in markdown format to the [proposals directory](./proposals).
+   Make sure to use the [template](0000-00-00_template.md) as the guide for what sections should be present in the
+   document. Put the creation date (the day you started preparing this design document) as the prefix and some unique
+   name as the suffix in the file name. Once the PR is proposed, a maintainer will assign a `proposal` label.
```

Comment:
it's not clear what the resulting file should be named. I assume `0000-00-00` is `YYYY-MM-DD`? Does it make sense to track? What date would that be, date of merge/acceptance or initial creation?

Reply:
From the README, it currently says:
> Put the creation date (the day you started preparing this design document)

I think that makes sense as it is a way to track how long the effort has been going one.

Reply:
Ah, must've missed that. LGTM